apps/coscrad-frontend/src/app/build-routes.tsx

```diff
@@ -66,6 +67,13 @@ export const buildRoutes = (contentConfig: ConfigurableContent): CoscradRoute[]
                 element: <NoteIndexContainer />,
             }),
         ],
+        [
+            !isNullOrUndefined(noteIndexToDetailConfig) && shouldEnableWebOfKnowledgeForResources,
+            () => ({
+                path: `${notesRoute}/:id`,
+                element: <NoteDetailPageContainer />,
```

Comment:
Awesome

Reply:
Not sure how this was missing. I need to return to this a bit more carefully sometime.